carts/views.py

```diff
@@ -32,3 +32,32 @@ def get(self, request):
 
         except Cart.DoesNotExist:
             return JsonResponse({"message" : "CART_NOT_EXIST"}, status = 404)
+    
+    @login_decorator
+    def patch(self, request):
+        try:
+            cart_id  = request.GET.get('cart_id')
+            data     = json.loads(request.body)
+            quantity = data['quantity']
+
+            if cart_id:
+                Cart.objects.filter(id = int(cart_id)).update(quantity = quantity)
+        
+            return JsonResponse({"message" : "SUCCESS"}, status = 201)
```

Comment:
```suggestion
            return JsonResponse({"message" : "SUCCESS"}, status = 200)
```
201은 새로운 resource가 생겼을 때 사용하므로 200이 더 적절할 것 같습니다.

Reply:
넵